src/argparse/arguments/Argument.java

```diff
@@ -175,8 +175,8 @@ public boolean argEquals(Argument arg) {
     @Override
     public String toString() {
         if (alias != null) {
-            return String.format("%s, %s\t\t%s", alias, token, help);
+            return String.format("  %s, %s\t\t%s", alias, token, help);
```

Comment:
One change I wanted to touch on is this. I think adding the spaces in front of the help messages is personal preference, and I prefer no spaces in the help menu. For that reason I reverted the change. However, if you prefer to have the spacing I think it should be inserted in the ArgumentParser.printHelp method. I have seen other argument parsers (python argparse I believe does this) have a help formatter class that you can subclass and customize the look and behavior of the help.